### DISCUSSION TITLE
Tutorial +project or first tutorial,what will be best for starter

### DISCUSSION THREAD

**Discussion by rakeshdeka** [EddieHubCommunity/support#2248] [Q&A]
Currently, I am learning web dev, which will be best for me. Because when I try to build projects I always get lost in the documentation.

**Comment by srisatya12** [comment]
@rakeshdeka Could you elaborate more? Like where you are getting stuck. Is it in CSS or Js?

**Reply by rakeshdeka** [reply]
@SriSatyaT  I was just asking about what should be my approach when learning something, should I build a project when learning about the tech or I can build it later?
